src/Summoner/Template/Cabal.hs

```diff
@@ -29,34 +30,34 @@ cabalFile Settings{..} = File (toString settingsRepo ++ ".cabal") cabalFileConte
 
     -- TODO: do something to not have empty lines
     cabalHeader :: Text
-    cabalHeader =
-        [text|
-        cabal-version:       2.0
-        name:                $settingsRepo
-        version:             0.0.0
-        synopsis:            $settingsDescription
-        description:         $settingsDescription
-        $githubHomepage
-        $githubBugReports
-        license:             $licenseName
-        license-file:        LICENSE
-        author:              $settingsFullName
-        maintainer:          $settingsEmail
-        copyright:           $settingsYear $settingsFullName
-        category:            $settingsCategories
-        build-type:          Simple
-        extra-doc-files:     README.md
-                           , CHANGELOG.md
-        tested-with:         $testedGhcs
-        |]
+    cabalHeader = T.unlines $
+        [ "cabal-version:       2.0"
+        , "name:                " <> settingsRepo
+        , "version:             0.0.0"
+        , "synopsis:            " <> settingsDescription
+        , "description:         " <> settingsDescription
+        ] ++
+        [ githubHomepage   | settingsGitHub ] ++
```

Comment:
What do you think about moving `homepage` part with alignment here to make it more clearer about indentation?

```haskell
[ "homepage:            " <> githubUrl | settingsGitHub ] ++
```